windows_installer/wireguard.wxs

```diff
@@ -112,7 +112,7 @@
 				<ServiceControl Id="DummyService.3AA0C492_29F4_4342_B608_DB95B2DECB13" Name="DummyService.3AA0C492_29F4_4342_B608_DB95B2DECB13" /><!-- A dummy to make WiX create ServiceControl table for us. -->
 			</Component>
 			<Component Directory="SystemFolder" Win64="no" Id="Wg32Executable" Guid="5ca31841-97d8-4614-a318-f1e268135ba7">
-				<File Source="..\x86\wg.exe" Id="Wg32Executable" />
+				<File Source="..\amd64\wg.exe" Id="Wg32Executable" />
```

Comment:
Revert this change, I'll need to fix x86 builds on my end as I'm skipping them for faster builds but that here will use the wrong binary for x86